prime-router/src/main/kotlin/azure/ReportFunction.kt

```diff
@@ -258,7 +258,15 @@ class ReportFunction : Logging {
 
         // queue messages here after all task / action records are in
         actionHistory.queueMessages(workflowEngine)
-        return response
+        val uri = request.getUri()
+        responseBuilder.header(
+            HttpHeaders.LOCATION,
```

Comment:
Adds a header to inform the user where to find updates about their data. This could be removed for now if we're not ready for any sender to use this api.